examples/reminderbot/domain.yml

```diff
@@ -17,19 +14,25 @@ intents:
 - EXTERNAL_reminder:
     triggers: action_react_to_reminder
 - EXT_reminder
+
 entities:
 - name
 - plant
+
 responses:
   utter_what_can_do:
   - text: What can I do for you?
+
   utter_goodbye:
   - text: Bye
+
 actions:
 - action_set_reminder
 - action_forget_reminders
 - action_react_to_reminder
 - action_tell_id
 - action_warn_dry
-- utter_what_can_do
```

Comment:
Why were these deleted?

Reply:
This new example bot wasn't merged yet when these changes were made https://github.com/RasaHQ/rasa/pull/5083/files